oie/ja-jp/content/resources/scripts/js/ignore-dompurify.js

```diff
@@ -13,17 +13,3 @@ requirejs.onError = function (err) {
   }
   throw err;
 };

```

Comment:
What is the status for lower case redirects? I see we are removing the function, so just want to know why we don't need it anymore

Reply:
we actually need it, I'll update LL branch